Update a few phrases for mise install (#1712)
jdx/mise: docs/cli-reference.md

```text
Install a tool version

> `install` will install a tool version to `~/.local/share/mise/installs/<PLUGIN>/<VERSION>`

Reviewer: ```suggestion
Installs a tool version to `~/.local/share/mise/installs/<PLUGIN>/<VERSION>`
```

> Tools are inactive without being enabled, however.

Reviewer: ```suggestion
Installing alone will not activate the tools so they won't be in PATH.
```

I'm a little uncomfortable with this language since it still will create shims, so shim users _will_ have it in PATH but the shims won't work. I think that this language does make sense for the majority of users though and that subtle behavior difference won't make any practical difference since the way shims behave is if they're not active it falls back to the system bin (if any). So it behaves basically as if it "wasn't in PATH".

Author: Good point!

> To enable a tool, set up a `.mise.toml`/`.tool-version` file or `mise use`.

Reviewer: ```suggestion
To install and/or activate in one command, use `mise use` which will create a `.mise.toml` file in the current directory to activate this tool when inside the directory.
```

> Another way is to use a tool explicitly with `mise exec <TOOL>@<VERSION> -- <COMMAND>`.

Reviewer: ```suggestion
Alternatively, run `mise exec <TOOL>@<VERSION> -- <COMMAND>` to execute a tool without creating config files.
```

          Directly pipe stdin/stdout/stderr from plugin to user Sets --jobs=1

  -v, --verbose...

>           Show installation output. This argument will print plugins' logs.
>           When plugin downloads assets, mise will show download logs.
>           When plugin configures and compiles tools, mise will show configuration and compilation logs.

Reviewer: ```suggestion
          Show installation output.
          
          This argument will print plugin output such as download, configuration, and compilation output.
```
